components/Storyblok/SbBasicCard.tsx

```diff
@@ -14,6 +14,7 @@ import { type SbImageType } from './Storyblok.types';
 export type SbBasicCardProps = {
   blok: {
     _uid: string;
+    id: string;
```

Comment:
I vaguely remember something about the `id` in storyblok's response not being a good fit for an HTML element ID. I am trying to remember if that was because it was not unique or if it was because the contents were not a valid id name...

Perhaps it was both. 

I'd have to look at the API/docs to find out but perhaps you swap this out for a `useId()` call instead?

Reply:
@sherakama Hmmm I don't seem to remember the id field causing problems. We still have it for our other Storyblok sites 👀  I did add a regex check to not let the content author use the id "main-content" (that was I think Alumni) so I can add that check in Storyblok for this site too. 

useId doesn't work well here because the content author will add the id manually when they want to link to a section (from a newsletter for example).

Reply:
Oh I see what you were thinking @sherakama - these are actually new id fields that I manually added to the Storyblok components for the content authors to input themselves, so not the ids that are automatically associated with each components in the Storyblok response 🫡

Reply:
Oh, you created `id` fields? Sounds risky but you have power users so... ok.

Reply:
Yes. I added instructions and regex check 😆